src/Features/Core/Portable/InlineHints/AbstractInlineParameterNameHintsService.cs

```diff
@@ -253,5 +259,29 @@ static bool SuffixMatchesParameterName(ReadOnlyMemory<char> suffix, string param
                        suffix.Span[1..].Equals(parameterName.AsSpan()[1..], StringComparison.Ordinal);
             }
         }
+
+        private bool ParameterMatchesArgumentName(SyntaxNode argument, IParameterSymbol? parameter, ISyntaxFactsService syntaxFacts)
+        {
```

Comment:
if (parameter == null)
    return false;

(when do you not have a parameter?)

Reply:
I was following the pattern above. At this point the param shouldn't be null

Reply:
if it shoud'nt be null, then why is it `?` and why is there a null check below? :)

Reply:
> if (parameter == null)
> return false;

because I assumed this meant you wanted a null check? I will remove it